src/main/java/com/simpletalk/enterprise/SimpleTalkController.java

```diff
@@ -24,15 +24,23 @@ public class SimpleTalkController {
 
     /**
      * Handle the / endpoint
+     *
      * @return
      */
     @RequestMapping("/")
     public String index(Model model) {
+        Post post = new Post();
```

Comment:
populating the post class to enable model.addAttribute funciton

Reply:
This looks like test data, which should likely live in a stub class, perhaps on the service layer.  I wouldn't hardcode this into the production controller class.